mifosng-android/src/main/java/com/mifos/mifosxdroid/adapters/ClientNameListAdapter.java

```diff
@@ -16,6 +16,8 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
```

Comment:
Its not a big deal but from next time make sure to remove unnecessary blank lines between the import statment it makes file smaller. ;)
PS: No need to send commit for this PR

Reply:
I will take care of it :)